en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,143 @@
+- help community modules add support for the new major release early on.
+
+Once a new major enters the beta period, it should be compatible with the dependencies intended to ship in the stable release. Substantial breaking changes should be avoided post-beta.
+
+Prior to the beta period, alpha releases are published at the discretion of the development team.
+
+### Stable release
+
+New major releases of Silverstripe CMS are tagged between April and June of odd years.
+
+### Active development
+
+Once a major release is stable, it enters a period of active development. A major release in active development receives:
+- regular minor releases that ships new feature and API in a backward compatible way
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+A major releases line stays in *active development* for approximatively two years. *Active development* for a major release line ends when the next major release is tagged stable.
+
+Only one major release line is in *active development* at any given time.
+
+### Bug and security fixes
+
+A Silverstripe CMS major release line enters the *bug and security fixes* phase once the follow major release is tagged stable.
+
+A major release line in the *bug and security fixes* phase receives:
+- regular patches for bugs at all impact levels
+- security patches for all vulnerabilities.
+
+A major release line in the *bug and security fixes* phase does **not** receive:
+- new feature
+- new APIs
+- new minor releases.
+
+A major release line stays in the *bug and security fixes* phase for 1 year.
+
+### Security fixes only
+
+At the end of *bug and security fixes* phase, a Silverstripe CMS major release line transitions to the *security fixes only* phase.
+
+In the *security fixes only* phase, a Silverstripe CMS major release line only receives patches for high impact security issues defined has having a CVSS of 7.0 or above.
+
+### End-of-life
+
+A Silverstripe CMS major release line reaches *end-of-life* once the next major release line exits active development.
+
+End-of-life major releases do not receive updates of any kind including security fixes or bug fixes.
+
+## PHP support commitments
+
+Silverstripe CMS major releases track PHP releases. 
+
+The Silverstripe CMS release cycle is built around these assumptions.
+
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years with a year of limited support. 
+
+At launch, a Silverstripe CMS major release supports all PHP versions in full support. PHP versions in limited support are not supported at launch by new Silverstripe CMS major release.
+
+Following the initial release of a Silverstripe CMS major release, the development team aims to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. CMS 5 should receive official support for an eventual PHP 8.3 in early 2024.
```

Comment:
```suggestion
Following the initial release of a Silverstripe CMS major version, the development team aims to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. CMS 5 should receive official support for an eventual PHP 8.3 in early 2024.
```
"Following the release of a release" sounds pretty bad.
An alternative would be "Following a new Silverstripe CMS Major release,"

Reply:
Switched to "Following the initial launch of a Silverstripe CMS major release"